containers/RoundSection/index.js

```diff
@@ -67,13 +25,80 @@ const RoundSection = ({
   btnStyle,
   sectionSubTitle,
   cardArea,
-  btnTitle,
+  btnTitle
 }) => {
+  const [hasRound, setHasRound] = useState(true);
+  const [endTime, setEndTime] = useState(0);
+  const [isLoading, setLoading] = useState(true);
+
+  async function fetchData() {
+    const res = await fetch("https://api.acquity.io/v1/round/active");
+    if (res!==null){
+    res
+      .json()
+      .then(res => {
+        if (!res["is_concluded"]) {
+          setEndTime(res["end_time"]);
+        }else{
+          setHasRound(false);
+        }
+      })
+      .then(setLoading(false));
+    }else{
+      setHasRound(false);
+      setLoading(false);
+    }
+  }
+
+  useEffect(() => {
+    fetchData();
```

Comment:
Nit: we can actually inline the whole `fetchData()` function in this useEffect function, then call `fetchData()`:
Notnit: You should add an empty array `[]` as the dependency of useEffect so it only runs once on render. See https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects for more info
```
useEffect(() => {
    async function fetchData() { ...}

   fetchData();
}, []); // <-- add an empty array so it only loads ONCE on render, this is equivalent to React class' componentDidMount
```